lxc/image.go

```diff
@@ -229,8 +241,8 @@ func (c *cmdImageCopy) Run(cmd *cobra.Command, args []string) error {
 		imgInfo.Public = true
 	} else {
 		// Resolve any alias and then grab the image information from the source
-		image := c.image.dereferenceAlias(sourceServer, imageType, name)
-		imgInfo, _, err = sourceServer.GetImage(image)
+		var err error
```

Comment:
is there a need to define a new err here rather than reusing the earlier one above?

Reply:
fixed